notificationworkerlambda/src/main/scala/com/gu/notifications/worker/models/SendingResults.scala

```diff
@@ -41,3 +44,45 @@ case class PerformanceMetrics(
   chunkTokenSize: Int,
 )
 
+case class LatencyMetricsForCloudWatch(uniqueValues: List[Long], orderedCounts: List[Int])
+
+object LatencyMetrics {
```

Comment:
Some of the logic here is a little complicated - it's unit tested [here](https://github.com/guardian/mobile-n10n/blob/6090beda81fe2a4cfd9634abda362f5d592b95b5/notificationworkerlambda/src/test/scala/com/gu/notifications/worker/models/SendingResultsSpec.scala).

Reply:
Thanks - the tests do make it clear what's going on :)